File: app/src/main/java/fuck/wifiadbtoggle/droidvendorssuck/ScheduleAlarmReceiver.java
package fuck.wifiadbtoggle.droidvendorssuck;

import android.content.BroadcastReceiver;
import android.content.Context;
import android.content.Intent;

public class ScheduleAlarmReceiver extends BroadcastReceiver {
    @Override
    public void onReceive(Context context, Intent intent) {
        ScheduleManager.applyScheduleNow(context);
        ScheduleAlarmScheduler.scheduleNext(context);
    }
}

File: app/src/main/java/fuck/wifiadbtoggle/droidvendorssuck/ScheduleAlarmReceiver.kt
package fuck.wifiadbtoggle.droidvendorssuck

import android.content.BroadcastReceiver
import android.content.Context
import android.content.Intent

class ScheduleAlarmReceiver : BroadcastReceiver() {
    override fun onReceive(context: Context, intent: Intent?) {
        ScheduleManager.applyScheduleNow(context)
        ScheduleAlarmScheduler.scheduleNext(context)
    }
}
